client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -18,11 +87,91 @@ export default function PaymentMethodAdd(): ReactElement {
 				<CardHeading size={ 36 }>{ translate( 'Payment Methods' ) }</CardHeading>
 			</div>
 
-			<Card className="payment-method-add__body">
-				<div className="payment-method-add__body-left"></div>
+			<CheckoutProvider
+				onPaymentComplete={ () => {
+					onPaymentSelectComplete( {
+						successCallback: () => page( '/partner-portal/payment-methods/' ),
+						translate,
+						showSuccessMessage,
+						reloadSetupIntentId,
+					} );
+				} }
+				paymentMethods={ paymentMethods }
+				paymentProcessors={ {
+					card: ( data: unknown ) =>
+						assignNewCardProcessor(
+							{
+								useAsPrimaryPaymentMethod,
+								translate,
+								stripe,
+								stripeConfiguration,
+								stripeSetupIntentId,
+								cardElement: elements?.getElement( CardElement ) ?? undefined,
+								reduxDispatch,
+							},
+							data
+						),
+				} }
+				initiallySelectedPaymentMethodId="card"
+				isLoading={ isStripeLoading }
+			>
+				<Card className="payment-method-add__wrapper">
+					<CardHeading>{ translate( 'Credit card details' ) }</CardHeading>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Stored Credit Card Details' )` *ES Score: 9*